<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleFormDialog</class>
 <widget class="QDialog" name="GoogleFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="expressRadio">
     <property name="text">
      <string>Express Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="customRadio">
     <property name="text">
      <string>Custom Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="titleInput"/>
   </item>
   <item>
    <widget class="QLabel" name="descLabel">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="descInput"/>
   </item>
   <item>
    <widget class="QPushButton" name="continueBtn">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>